installer/windows/build_installer.py

```diff
@@ -0,0 +1,348 @@
+    massastation_server_url = "https://github.com/massalabs/thyra/releases/latest/download/thyra-server_windows_amd64"
+    massastation_app_url = "https://github.com/massalabs/Thyra-Menu-Bar-App/releases/latest/download/ThyraApp_windows-amd64.exe"
+
+    download_file(acrylic_url, ACRYLIC_ZIP)
+    download_file(massastation_server_url, SERVER_BINARIES)
+    download_file(massastation_app_url, APP_BINARIES)
+
+    move_binaries()
+
+    create_wxs_file()
+
+    # Build installer
+    try:
+        subprocess.run(
+            [
+                os.path.join(WIX_DIR, "candle.exe"),
+                "-ext",
+                "WixUIExtension",
+                "-ext",
+                "WixUtilExtension",
+                "-out",
+                os.path.join(BUILD_DIR, "config.wixobj"),
+                os.path.join(BUILD_DIR, "config.wxs"),
+            ],
+            check=True,
+        )
+        subprocess.run(
+            [
+                os.path.join(WIX_DIR, "light.exe"),
+                "-ext",
+                "WixUIExtension",
+                "-ext",
+                "WixUtilExtension",
+                "-sval",
+                "-out",
+                f"massastation_{VERSION}_amd64.msi",
+                os.path.join(BUILD_DIR, "config.wixobj"),
+            ],
+            check=True,
+        )
+    except subprocess.CalledProcessError as err:
+        print("Error building installer: ", err)
+        sys.exit(1)
+    except err:
+        print(err)
+        sys.exit(1)
+
+
+def install_dependencies():
+    """
+    Install dependencies if they are not already installed.
+
+    The main dependency is the WiX Toolset.
+    """
+
+    if os.path.exists(WIX_DIR):
+        return
+
+    wixtoolset_url = (
+        "https://wixdl.blob.core.windows.net/releases/v3.14.0.6526/wix314-binaries.zip"
+    )
+
+    download_file(wixtoolset_url, "wixtoolset.zip")
+
+    os.mkdir(WIX_DIR)
+    with zipfile.ZipFile("wixtoolset.zip", "r") as zip_ref:
+        zip_ref.extractall(WIX_DIR)
+    os.remove("wixtoolset.zip")
+
+
+if __name__ == "__main__":
+    # pylint: disable-next=pointless-string-statement
+    """
+    This is the main function of this script.
+
+    It checks if the script is being run on Windows and if so,
+    it installs the dependencies and builds the installer.
+    """
+
+    parser = argparse.ArgumentParser(
+        description="Build the MassaStation installer for Windows."
+    )
+    parser.add_argument(
+        "-v",
+        "--version",
+        action="store",
+        dest="version",
+        default=VERSION,
+        help="Set the version of the installer and MassaStation.",
+    )
+    parser.add_argument(
+        "-f",
+        "--force",
+        action="store_true",
+        dest="force_build",
+        default=FORCE_BUILD,
+        help="Force the build on non-Windows platforms.",
+    )
+    args = parser.parse_args()
+
+    VERSION = args.version
```

Comment:
it might not be a good practice to modify global variable, I would have pass the value to the functions build_installer and create_wxs_file